HISTORY.md

```diff
@@ -1,5 +1,9 @@
 # Version History
 
+## 1.1.2 / 2021-09-07
+
+- Updated dependencies
+
```

Comment:
I might point out the streamsview collection creation in the library, although I feel like it's optional since this sample is simply adjusting to the library's breaking change. i think it's fine either way, but personally I'd add it